packages/react-table/src/components/Table/utils/decorators/treeRow.tsx

```diff
@@ -61,6 +66,20 @@ export const treeRow = (onCollapse: OnTreeRowCollapse, onCheckChange?: OnCheckCh
             </span>
           )}
           {text}
+          {!!onToggleRowDetails && (
+            <span className={css(stylesTreeView.tableTreeViewDetailsToggle)}>
+              <Button
+                variant="plain"
+                aria-expanded={isDetailsExpanded}
+                aria-label={showDetailsAriaLabel || 'Show row details'}
+                onClick={event => onToggleRowDetails && onToggleRowDetails(event, rowIndex, content, rowData)}
+              >
+                <span className="pf-c-table__details-toggle-icon">
+                  <EllipsisHIcon />
```

Comment:
does this need `aria-hidden`?

```suggestion
                  <EllipsisHIcon aria-hidden />
```